src/types.ts

```diff
@@ -6,5 +6,10 @@ export type AResult = {
   robust: number;
   robustDeviation: number;
   highLimit: number;
-  lowLimit: number;  
-};
\ No newline at end of file
+  lowLimit: number;
+};
+export type ReferenceResult = {
+  value: number;
+  method: string;
+} | null;
```

Comment:
Did this get mixed into this PR from the other branch?

Reply:
There was conflict with stash. I'm removing it now.